rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/IntegrationSteps.java

```diff
@@ -0,0 +1,116 @@
+package io.syndesis.qe.rest.tests.integrations;
+
+import org.assertj.core.api.Assertions;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+import io.syndesis.common.model.action.Action;
+import io.syndesis.common.model.connection.Connection;
+import io.syndesis.common.model.connection.Connector;
+import io.syndesis.common.model.extension.Extension;
+import io.syndesis.common.model.integration.Step;
+import io.syndesis.common.model.integration.StepKind;
+import io.syndesis.qe.bdd.AbstractStep;
+import io.syndesis.qe.bdd.entities.StepDefinition;
+import io.syndesis.qe.bdd.storage.StepsStorage;
+import io.syndesis.qe.endpoints.ConnectionsEndpoint;
+import io.syndesis.qe.endpoints.ConnectorsEndpoint;
+
+import io.syndesis.qe.endpoints.ExtensionsEndpoint;
+import io.syndesis.qe.utils.TestUtils;
+
+public class IntegrationSteps extends AbstractStep {
+    @Autowired
+    private StepsStorage steps;
+
+    @Autowired
+    private ConnectionsEndpoint connectionsEndpoint;
+
+    @Autowired
+    private ConnectorsEndpoint connectorsEndpoint;
+
+    @Autowired
+    private ExtensionsEndpoint extensionsEndpoint;
+
+    @When("^add \"([^\"]*)\" endpoint with connector's id \"([^\"]*)\" and \"([^\"]*)\" action and with:$")
```

Comment:
maybe change "connector's id" to "connector id"

Reply:
and change "with:$" to "with properties:$"

Reply:
changed